mdata/cache/accnt/flat_accnt.go

```diff
@@ -231,6 +254,11 @@ func (a *FlatAccnt) eventLoop() {
 	}
 }
 
+// // totalUsed returns the sum of cacheSizeUsed, cacheOverheadChunk, cacheOverheadFlat, and cacheOverheadLru
+// func (a *FlatAccnt) totalUsed() uint64 {
```

Comment:
i suspect that's not supposed to get merged, right?

Reply:
Yeah, I forgot that was in there. I had some plans for it, but then ended up not using it. I will take care of that.